openedx_webhooks/lib/github/test/models/test_github_webhook_event.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import arrow
+import pytest
+
+from openedx_webhooks.lib.github.models import GithubWebHookEvent
```

Comment:
Maybe it's not a big problem, but it seems unfortunate that in some places we have
```
from ..lib.github.models import GithubWebHookEvent
```
and in other places we have
```
from openedx_webhooks.lib.github.models import GithubWebHookEvent
```